operations/windows-cell.yml

```diff
@@ -51,6 +52,11 @@
               ca_cert: "((diego_bbs_client.ca))"
               client_cert: "((diego_bbs_client.certificate))"
               client_key: "((diego_bbs_client.private_key))"
+            consul:
+              ca_cert: "((consul_agent.ca))"
```

Comment:
Please change this to `diego_consul_client.ca` to match the current pattern where we reference the CA via the name of the certificate directly rather than configuring a CA that we "know" to be the same.